sources/packages/api/src/route-controllers/utils/address-utils.ts

```diff
@@ -0,0 +1,34 @@
+import { AddressDetailsModel } from "../../services";
+import { AddressDetailsAPIOutDTO } from "../models/common.dto";
+
+/**
+ * Util to transform address details for formIO.
+ * @param addressDetails  is the object that has
+ * address details of type AddressDetailsModel.
+ * @returns address info as AddressDetailsAPIOutDTO.
```

Comment:
As we discussed in the past, we must avoid using object type name for returns.

Reply:
this is a different scenario, please go through the comment if you didn't get it. let me know

Reply:
What I meant is we must avoid using AddressDetailsAPIOutDTO. for e.g. it can be written as address info as DTO/DTO object instead of using the object name.